service/src/main/java/crawlercommons/urlfrontier/service/rocksdb/RocksDBService.java

```diff
@@ -699,13 +699,13 @@ protected long deleteLocalCrawl(String crawlID) {
                 if (samePrefix) {
                     if (startKey == null) {
                         startKey =
-                                (prefixed_queue.getCrawlid().replaceAll("_", "%5F") + "_")
+                                (prefixed_queue.getCrawlid().replace("_", "%5F") + "_")
```

Comment:
am pretty sure we had `replaceAll` for a reason. why that change?

Reply:
Yep, the reasoning is valid. We should use `replace(...)` if no regex is involved.